OmiseSwift/API Models/Charge.swift

```diff
@@ -190,7 +190,7 @@ extension Charge {
         case ipAddress = "ip"
         case dispute
         case returnURL = "return_uri"
-        case authorizedURL = "authorized_uri"
+        case authorizedURL = "authorize_uri"
```

Comment:
I noticed that actually in our charge response it is `authorize_uri`  not `authorized_uri` , not sure if this is typo or intentional but this will result in the authorize_url not mapping and create nil in the test, can I have some confirm about this? @igroomgrim @tobias90

Reply:
Good catch 👍 
Checked, it use `authorize_uri` for the key mapping.
I think it should change the CodingKey case name to `authorizeURL` too.
Follow the spec from: https://www.omise.co/charges-api.

But if you update the CodingKey case name, you should update the `authorizedURL` property name in Charge model too. It would be good if you split out of this case to other PR. 🙇

Reply:
https://github.com/omise/omise-swift/pull/103
kindly review 🙏 @tobias90 @igroomgrim